src/js/shared/RemotePontoon.js

```diff
@@ -71,7 +71,7 @@ class RemotePontoon {
      * @public
      */
     getTeamBugsUrl() {
-        return `${this._baseUrl}/${this._team}/bugs/`;
+        return `${this._baseUrl}/${this._team}/bugs/?utm_source=pontoon-tools`;
```

Comment:
That 404s sadly.

Reply:
Yes, that's a bug in how Pontoon handles dashboard tab URLs.

Reply:
Fixed now.